dirsrvtests/tests/suites/password/pwdPolicy_attribute_test.py

```diff
@@ -796,6 +929,12 @@ def test_global_tpr_delayValidFrom_1(topology_st, create_user, request):
     assert our_user.get_attr_val_utf8('pwdTPRReset') == 'TRUE'
     now = time.mktime(time.gmtime())
     log.info("compare pwdTPRValidFrom (%s) vs now (%s)" % (our_user.get_attr_val_utf8('pwdTPRValidFrom'), time.gmtime()))
+    time.sleep(.5)
+    log.info("compare pwdTPRValidFrom (%s) vs now (%s)" % (our_user.get_attr_val_utf8('pwdTPRValidFrom'), time.gmtime()))
+    time.sleep(1)
+    log.info("compare pwdTPRValidFrom (%s) vs now (%s)" % (our_user.get_attr_val_utf8('pwdTPRValidFrom'), time.gmtime()))
+    time.sleep(2)
+    log.info("compare pwdTPRValidFrom (%s) vs now (%s)" % (our_user.get_attr_val_utf8('pwdTPRValidFrom'), time.gmtime()))
     assert (gentime_to_posix_time(our_user.get_attr_val_utf8('pwdTPRValidFrom'))) >= (now + ValidFrom - 2)
```

Comment:
Also, I see that the next test fails:
```
>       assert (gentime_to_posix_time(our_user.get_attr_val_utf8('pwdTPRValidFrom'))) >= (now + ValidFrom - 2)
...
        time_tuple = (int(gentime[:4]),
>                     int(gentime[4:6]),
                      int(gentime[6:8]),
                      int(gentime[8:10]),
                      int(gentime[10:12]),
                      int(gentime[12:14]),
                      0, 0, 0)
E       ValueError: invalid literal for int() with base 10: ''
```

And in the logs it's written that **pwdTPRValidFrom == -1**

Reply:
Yes, those tests are failing because of local password policy leftover (from test_local_tpr_maxuse_5). I am testing a fix for those failure